lib/helpers/printJestWarning.js

```diff
@@ -9,7 +9,10 @@ if (typeof jest !== 'undefined' && typeof window !== 'undefined') {
     'https://mongoosejs.com/docs/jest.html');
 }
 
-if (typeof jest !== 'undefined' && process.nextTick.toString().indexOf('nextTick') === -1) {
+if (typeof jest !== 'undefined' && process.nextTick.toString() ===
+  'function () {\n' +
```

Comment:
Unfortunately this check is way too brittle for us to merge. This would break even if Jest changes their internal code formatting a little bit.

We'll see if there is a way we can pull Jest's fake timer implementation without adding an extra `require()`.